Resources/translations/FSiAdminSecurity.en.yml

```diff
@@ -26,7 +26,7 @@ admin:
         email: Email
         button:
           send: Send me instructions
-      mail_sent: Reset password instructions sent
+      mail_sent_if_correct: If the address you have provided was corrent, then you should receive reset password instructions
```

Comment:
If the provided address was correct, you should receive instructions on resetting your password.